src/analysis/index.js

```diff
@@ -73,6 +73,10 @@ function translateChildren (deps, node, samples, appName, root = false) {
   }
 
   for (const c of node.children) {
+    if (
```

Comment:
instead of reducing the branches value, I would trace the ignored ones

```suggestion
    /* istanbul ignore if */
    if (
```

Reply:
good point

Reply:
Actually, it needs to be readjusted because we don't cover 90% in all cases (depending on OS) even without this PR. It's visible now because we enabled the actions in PR's.